libs/api-interfaces/src/lib/aggregate-views/view-models/resources/media-items/media-item.view-model.interface.ts

```diff
@@ -6,9 +6,9 @@ export interface IMediaItemViewModel extends IBaseViewModel {
 
     titleEnglish?: string;
 
-    url: string;
-
     mimeType: MIMEType;
 
     lengthMilliseconds: number;
+
+    url: string;
```

Comment:
Note that this is simply built in the query service now and doesn't have to be on the view model itself.